frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -224,7 +217,7 @@ const getOperandStatus = (obj: K8sResourceKind): OperandStatusType => {
 export const OperandStatus: React.FunctionComponent<OperandStatusProps> = ({ operand }) => {
   const status: OperandStatusType = getOperandStatus(operand);
   if (!status) {
-    return <div className="text-muted">Unknown</div>;
+    return <div className="text-muted">-</div>;
```

Comment:
I don't think we use `text-muted` with `-` in other places.

```suggestion
    return <>-</>;
```

Reply:
https://github.com/openshift/console/blob/master/frontend/public/components/persistent-volume-claim.jsx#L134

Reply:
@jhadvig You've found another outlier. Most places we *don't* add `text-muted` to the `-`. 

https://github.com/openshift/console/blob/master/frontend/public/components/container.tsx#L308

https://github.com/openshift/console/blob/master/frontend/public/components/machine-autoscaler.tsx#L107

https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/pipelines/list-page/PipelineRow.tsx#L45

etc

Reply:
@jhadvig In fact, the PVC example you linked to was just added recently in #4482, and we have other columns that have `-` without `text-muted` in the same table. That's probably not the example we should use.

Reply:
`text-muted` removed from the PVC table in #5382